### ISSUE TITLE
white screen on mobile?

### ISSUE THREAD

**Issue by inspire22** [aaronksaunders/ionic-vue-split-screen#2]
Thanks for making this, I've been struggling trying to get a split-view that would work for a tablet version of my app.  

When I resize the example screen you linked it just goes blank for mobile widths - how to have it be a "normal" drawer nav when that is the case? https://codesandbox.io/s/ionic-vue-web-components-split-screen-qmv57  Then I click 'open in new window' for the app and turn on the chrome mobile device spoofing.

**Comment by inspire22** [comment]
I got it working.  It's *VERY* particular about what elements are in what other elements.  This page helped:
https://github.com/ionic-team/ionic/tree/master/core/src/components/split-pane

Things I was doing wrong:
- my sidebars were include next to my main-content header, rather than in App.vue 
- extra wrapper elements like divs break everything.
- "main" doesn't work, you need "id='main' then content-id='main' on the ion-split-pane <ion-split-pane content-id="main">

The last one is what your repo was missing, so it was still calling the center pane one of the sidebars, and hiding it when the screen was too narrow.  

TLDR - change 
<ion-page main class="ion-page"> to <ion-page id="main" class="ion-page">
and <ion-split-pane when="sm"> to <ion-split-pane when="sm" content-id="main">

Also for me adding style='flex-grow: 4' to the main panel made it grow more nicely based on screen-width.

Thanks!

**Comment by aaronksaunders** [comment]
sorry for late response, glad you resolved your issues
